src/platform/silabs/wifi/WifiInterface.h

```diff
@@ -71,6 +73,7 @@ typedef struct wfx_wifi_scan_result
     uint8_t bssid[kWifiMacAddressLength];
     uint8_t chan;
     int16_t rssi; /* I suspect this is in dBm - so signed */
```

Comment:
can you confirm the needed type and then remove the informal comment

Reply:
sure will take it up as a part of the cleanup activity